RecoTracker/FinalTrackSelectors/python/earlyGeneralTracks_cfi.py

```diff
@@ -57,7 +61,8 @@
         'pixelPairStepTracks',
         'mixedTripletStepTracks',
         'pixelLessStepTracks',
-        'tobTecStepTracks'
+        'tobTecStepTracks',
+        'siStripTripletStepTracks'
```

Comment:
uhmmm, I'm afraid we would need to add a new modifier for Run3
because we do not want to have new iterations run on Run2 by deafult

Reply:
why not?
In most of reco developments we attempt to apply the algorithms uniformly to all available data to the extent possible.
It seems more straightforward to add this everywhere.

Reply:
back-compatibility ?